rbx_dom_weak/src/brick_color.rs

```diff
@@ -18,6 +18,26 @@ macro_rules! make_brick_color {
 					_ => None,
 				}
 			}
+
+			pub fn as_rgb(self) -> [u8; 3] {
+				match self {
+					$(
+						BrickColor::$enum => [$color3_r, $color3_g, $color3_b],
+					)+
+				}
+			}
+
+			pub fn as_rgb_f32(self) -> [f32; 3] {
+				match self {
+					$(
+						BrickColor::$enum => [
+							($color3_r as f32) / 255.0,
+							($color3_g as f32) / 255.0,
+							($color3_b as f32) / 255.0,
+						],
+					)+
+				}
+			}

```

Comment:
Can we leave these APIs `pub(crate)` for now? I'm not sure on naming, like which one should be default, and whether it should be `as_color3` and `as_color3uint8` or whatever.